spigot-impl/src/main/java/me/aurium/beetle/spigot/SpigotBeetleFactory.java

```diff
@@ -31,7 +33,8 @@ public SpigotBeetleFactory(JavaPlugin plugin, boolean isDebug) {
      */
     public SpigotBeetle build() {
         //produce dependencies
-        SpigotTasker tasker = new SpigotTasker(plugin);
+        SpigotTasker tasker = new SpigotTasker(Executors.newFixedThreadPool(10000), plugin);
```

Comment:
You are way ahead of your time with another 10,000 threads. Loom is still in early access!